src/Take.Elephant.Kafka/KafkaSenderQueue.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace Take.Elephant.Kafka
 {
-    public class KafkaSenderQueue<T> : ISenderQueue<T>, IDisposable
+    public class KafkaSenderQueue<T> : IStreamSenderQueue<T>, IDisposable
     {
-        private readonly IProducer<Null, string> _producer;
+        private readonly IProducer<string, string> _producer;
```

Comment:
This is a breaking change, which may affect a lot of usages of the implementation.

I recommend making this in a separate class.

Reply:
I agree with you. I will close this pull request and make a new one with the implementation of the other class. I believe there is no need for this interfaces changes and heritages of this PR, so it's better to make a new one and abandon this